e2e/csi/csi.go

```diff
@@ -184,7 +191,14 @@ func (tc *CSIVolumesTest) TestEFSVolumeClaim(f *framework.F) {
 	// does not.
 	// this runs the equivalent of 'nomad job stop'
 	nomadClient.Jobs().Deregister(writeJobID, false, nil)
-	e2eutil.WaitForAllocStopped(t, nomadClient, writeAllocID)
+	// instead of waiting for the alloc to stop, wait for the volume claim gc run
+	require.Eventuallyf(func() bool {
+		vol, _, err := nomadClient.CSIVolumes().Info("ebs-vol0", nil)
```

Comment:
We should use `volID` in both tests and not the string literal. In this case, this is the the EFS test so it's the wrong volume as a result.

Reply:
yikes, thanks